cvprac/cvp_api.py

```diff
@@ -3800,6 +3840,15 @@ def get_roles(self):
         url = '/role/getRoles.do?startIndex=0&endIndex=0'
         return self.clnt.get(url, timeout=self.request_timeout)
 
+    def delete_role(self, rolekey):
+        ''' Remove specified role from CVP
+            Args:
+                rolekey (str): role key on CVP
+        '''
+        data = [rolekey]
+        return self.clnt.post('/role/deleteRoles.do', data=data,
+                              timeout=self.request_timeout)
+
```

Comment:
Let's create a function for delete_roles() that takes a list of role keys and then another function for delete_role that will take one role key as string and call delete_roles() where the key is placed into a list.

Reply:
done. added tests for both delete_role and delete_roles.